org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/reconciler/XtextDocumentReconcileStrategy.java

```diff
@@ -191,4 +188,10 @@ public boolean isCanceled() {
 		}
 	}
 	
+	/**
+	 * @since 2.14
+	 */
+	protected void addStrategy(IReconcilingStrategy strategy) {
+		strategyList.add(strategy);
```

Comment:
who calls this? do we make sure add and iterate dont happen at the same time?

Reply:
This is called by the subclass that is bound when using code mining